packages/driver/src/cy/commands/origin/index.ts

```diff
@@ -64,6 +64,7 @@ export default (Commands, Cypress: Cypress.Cypress, cy: Cypress.cy, state: State
         name: 'origin',
         type: 'parent',
         message: urlOrDomain,
+        timeout: 0,
```

Comment:
This is an incidental fix. I noticed that `cy.origin()` has a visual timer bar that goes across it in the command log (regardless of `cy.pause()` usage, but particularly noticeable with `cy.pause()`. The log for `cy.origin()` shouldn't have a timeout anyway since the command itself does not have a timeout.